transactions/set-approved-id-list/2025/feb-4/README.md

```diff
@@ -0,0 +1,22 @@
+# Add T-systems execution node to the approved list of node
+
+> Feb 4th, 2025
+
+Transaction: [add_approved_id_list.cdc](../../../../templates/add_approved_id_list.cdc)
```

Comment:
@joshuahannan - I realized that the node IDs that are unstaked are automatically removed from the approved list.
` flow scripts execute get_approved_nodes.cdc -n mainnet | grep 2b396b7fab0102f104a2af7e095b145cc14da28f863564802e158afc3e07e638 ` returns nothing.
Hence, my earlier transaction to replace will not work as it paniced when the node id to remove was not found.
Reverting back to the old way of adding node using the existing `add_approved_id_list.cdc` transaction.

Reply:
sounds good to me